pages/api/availability/eventtype.ts

```diff
@@ -61,13 +62,13 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     };
 
     if (req.method == "POST") {
-      await prisma.eventType.create({
+      const event_type = await prisma.eventType.create({
```

Comment:
minor: eventType

Q: If we are going to add the created resource in the success 201 response shall we go all the way - remove the message entirely? (Maybe add a location header if possible)

Reply:
IMHO a proper create endpoint is one which returns the created object. Same for update.

Reply:
Sorry I mean the literal `message` - when you return the full created object the success is kinda given; ipv:

```javascript
- res.status(201).json({ message: "Event created successfully", event_type });
+ res.status(201).json(eventType);
```

Reply:
Ah, yes. I think it can be removed indeed.